docs/sources/getting-started/getting-started.md

```diff
@@ -1,58 +1,58 @@
+identifier = "getting_started-grafana"
 parent = "guides"
 weight = 200
 +++
 
-# Getting started
+# Getting started with Grafana
 
-This guide will help you get started and acquainted with Grafana. To learn more about Grafana in general, refer to [What is Grafana?]({{< relref "what-is-grafana.md" >}}).
+This topic helps you get started with Grafana and build your first dashboard. To learn more about Grafana, refer to [What is Grafana?]({{< relref "what-is-grafana.md" >}}).
 
-## Install Grafana
+## Step 1: Install Grafana
 
-This step varies according to your computer operating system. Refer to the instructions for your OS in the [Installation]({{< relref "../installation/_index.md" >}}) section for instructions.
+Grafana can be installed on many different operating systems. For a list of minimum hardware and software requirements, as well as instructions on installing Grafana, refer to [Install Grafana]({{< relref "../installation/_index.md" >}}).
 
-## Log in for the first time 
+## Step 2: Log in
 
-1. Open your web browser and go to http://localhost:3000/. `3000` is the default HTTP port that Grafana listens to if you haven’t configured a different port.
-1. On the login page, type `admin` for the username and password.
-1. Change your password. 
+To log in to Grafana for the first time:
 
-> **Note:** We strongly encourage you to follow Grafana best practices and change the default administrator password. Don't forget to record your credentials!
+1. Open your web browser and go to http://localhost:3000/. The default HTTP port that Grafana listens to is `3000` unless you have configured a different port.
+1. On the login page, enter `admin` for username and password.
+1. Click **Log In**. If login is successful, you will see a prompt to change the password.
+1. Click **OK** on the prompt, then change your password.
 
-## Create a dashboard
+> **Note:** We strongly recommend that you follow Grafana's best practices and change the default administrator password. Don't forget to record your credentials!
 
-1. Click **New dashboard**.
-1. Click **Add Query**. Grafana creates a basic graph panel with the Random Walk scenario.
-1. Save your dashboard. Click the **Save dashboard** icon in the top corner of the screen.
+## Step 3: Create a dashboard
 
- Congratulations, you have gotten started with Grafana! You have a dashboard and are displaying results. Feel free to experiment with what you have built, continue on to add another data source, or explore [Next steps](#next-steps).
+To create your first dashboard:
 
-## Next steps
+1. Click the **+** icon on the left panel, select **Create Dashboard**, then click **Add new panel**.
+1. In the New Dashbard/Edit Panel view, go to the **Query** tab.
+1. Configure your [query]({{< relref "../panels/queries.md" >}}) by selecting ``-- Grafana --`` from the [data source selector]({{< relref "../panels/queries.md/#data-source-selector" >}}). This generates the Random Walk dashboard.
```

Comment:
I don't think you mean dashboard here. Perhaps panel? Or graph?

Reply:
I went through the steps. It is dashboard.